l10n_it_abicab/abicab.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2012
+#    Associazione OpenERP Italia (<http://www.openerp-italia.org>)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#############################################################################
+
+from openerp import models, fields, api
+
+
+class ResBank(models.Model):
+
+    _inherit = "res.bank"
+
+    abi = fields.Char(size=5, string='ABI')
+    cab = fields.Char(size=5, string='CAB')
+
+
+class ResPartnerBank(models.Model):
+
+    _inherit = "res.partner.bank"
+
+    bank_abi = fields.Char(size=5, string='ABI')
+    bank_cab = fields.Char(size=5, string='CAB')
+
+    @api.onchange('bank')
+    @api.cr_uid_ids_context
```

Comment:
It would also work without them, but they allow the method to be called in a v8 manner.
For reference:
```
def cr_uid_ids_context(method):
    """ Decorate a traditional-style method that takes `cr`, `uid`, `ids`,
        `context` as parameters. Such a method::

            @api.cr_uid_ids_context
            def method(self, cr, uid, ids, args, context=None):
                ...

        may be called in both record and traditional styles, like::

            # recs = model.browse(cr, uid, ids, context)
            recs.method(args)

            model.method(cr, uid, ids, args, context=context)

        It is generally not necessary, see :func:`guess`.
    """
```

So, that's fine for me, thanks